readthedocs/oauth/services/github.py

```diff
@@ -28,7 +28,7 @@ class GitHubService(Service):
 
     adapter = GitHubOAuth2Adapter
     # TODO replace this with a less naive check
-    url_pattern = re.compile(r'github\.com')
+    url_pattern = re.compile(re.escape(GitHubOAuth2Adapter.web_url))
```

Comment:
I just realized that we can simply use `self.adapter` in all this class.

(it's the common usage for all the services implemented in RTD)